builder/parser/testfiles/json/Dockerfile

```diff
@@ -4,5 +4,5 @@ CMD ["a"]
 CMD ["a","b"]
 CMD [ "a", "b" ]
 CMD [	"a",	"b"	]
-CMD	[	"a",	"b"	]	
+CMD	[	"a",	"b"	]
```

Comment:
This one was _definitely_ on purpose (I know because I'm the one who put it here).